README.md

```diff
@@ -54,6 +54,7 @@ Visit [Amibot-tg](https://t.me/Achintya_test_bot) on telegram from there you can
 - `/attendance`: Get your attendance information.
 - `/schedule`: Get your class schedule.
 - `/about`: Get details about our project.
+- `/feedback`: Fill all your faculty feedback in one go.
```

Comment:
yah this looks good, add the wifi command as well

Reply:
done